src/js/components/Dashboard.js

```diff
@@ -205,10 +205,12 @@ class Dashboard extends Component {
       <div ref="content" className="dashboard">
         <Header direction="row" justify="between" large={true} pad={{horizontal: 'medium'}}>
           {title}
-          <Search ref="search" inline={true} className="flex"
-            placeHolder="Search"
-            defaultValue={searchText} onChange={this._onSearch}
-            suggestions={searchSuggestions} />
+          <Box direction="row" pad={!navActive ? {horizontal: 'medium'} : 'none'} className="flex">
```

Comment:
I think this will leave too large a margin between the title and Search placeholder text when the Search doesn't have focus. I'm thinking what we really should do is decrease the left padding inside the Search to small and put a pad={{between: 'small'}} on the Header.